src/Dumplie/Application/Query/Customer/CartQuery.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Query\Customer;
+
+use Dumplie\Application\Query\Customer\Result\Cart;
+
+interface CartQuery
+{
+    /**
+     * @param string $cartId
+     *
+     * @return Cart
+     */
+    public function getById(string $cartId) : Cart;
```

Comment:
@Arminek "find" can returns null (so you cannot use return type hint), "get" always returns requested object or throw exception. @norzechowicz if we throws exception from this method when cart is not found or something we should add "@throws" in php doc BTW

http://ocramius.github.io/doctrine-best-practices/#/95
http://ocramius.github.io/doctrine-best-practices/#/96